lib/shared_data/include/SharedDataTypes.h

```diff
@@ -44,7 +45,7 @@ struct PedalsSystemData_s
 struct DrivetrainDynamicReport_s
 {
     uint16_t measuredInverterFLPackVoltage;
-    float measuredSpeeds[NUM_MOTORS];
+    float measuredSpeeds[NUM_MOTORS]; // rpm
```

Comment:
eventually it would be nice if we did something similar here as to what was done with DB, i.e., type aliasing for physical units

`typedef float speed_rpm`